docs/resources/aws_elasti_cache_cluster.md

```diff
@@ -0,0 +1,92 @@
+---
+title: About the aws_elasti_cache_cluster Resource
```

Comment:
Would be worthwhile renaming these resources so there is parity with the terraform resource e.g. `aws_elasticache_cluster` 
https://www.terraform.io/docs/providers/aws/r/elasticache_cluster.html

Reply:
Ok, Stuart 👍